sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/src/Internals/Statsbeat/Statsbeat.cs

```diff
@@ -0,0 +1,49 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Diagnostics.Metrics;
+using System.Threading;
+using OpenTelemetry;
+using OpenTelemetry.Metrics;
+
+namespace Azure.Monitor.OpenTelemetry.Exporter.Internals
+{
+    internal static class Statsbeat
+    {
+        private static readonly Meter s_myMeter = new("AttachStatsBeatMeter", "1.0");
+
+        private const string StatsBeat_ConnectionString = "<StatsBeat_ConnectionString>";
+
+        // TODO: Move IsWindowsOS() to new class
+        // Do we need to support OSX?
+        internal static string s_OS = StorageHelper.IsWindowsOS() ? "windows" : "linux";
```

Comment:
if (RuntimeInformation.IsOSPlatform(OSPlatform.Linux)) why not this!

Reply:
Removing this to keep PR short. will revisit this in later PRs